src/test_workflow/benchmark_test/benchmark_test_suite_execute.py

```diff
@@ -0,0 +1,91 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import glob
+import json
+import logging
+import os
+import shutil
+import subprocess
+
+import pandas as pd
+
+from system.temporary_directory import TemporaryDirectory
+from test_workflow.benchmark_test.benchmark_args import BenchmarkArgs
+from test_workflow.benchmark_test.benchmark_test_base import BenchmarkTestBase
+
+
+class ExecuteTestSuite(BenchmarkTestBase):
+    def execute(self) -> None:
+        self.form_command(self.args)
+        log_info = f"Executing {self.command.replace(self.endpoint, len(self.endpoint) * '*').replace(self.args.username, len(self.args.username) * '*')}"
+        logging.info(log_info.replace(self.password, len(self.password) * "*") if self.password else log_info)
+        try:
+            subprocess.check_call(f"{self.command}", cwd=os.getcwd(), shell=True)
+            if self.args.cluster_endpoint:
```

Comment:
This is supposed to be `if self.args.cluster_endpoint or self.args.distribution_url:`. 
Can you please make sure the execute logic is not changed for existing code?